nlmpc:
  cost_weight:
    Q1v: 5
    Q2v: 3
    Pf1v: 50.0
    Pf2v: 30
    Rv: 1
  ph: 10
  ch: 5
  max_eval_num: 20
  filename1: "/home/siqihu/MasterThesis/Thesis/temp_data/me20.csv"
# goal: [0.4424, 0.4709, 0.5575, 0.9643, 0.0435, -0.2612, -0.0067]
# goal : [0.5911, 0.2160, 0.2377, -0.1757, 0.4291, 0.8531, 0.2393]
# goal : [0.2308, 0.1347, 0.5499, 0.9271, 0.3433, -0.1110, 0.1020]
# goal : [-0.2958, -0.2017, 0.5291, -0.1375, 0.7202, 0.2012, 0.6496]
# goal : [0.4941, 0.3911, 0.4779, -0.3510, 0.3688, -0.6909, -0.5133]
goal : [-0.16434678,  0.21842383,  0.54144014, 0.42083710476149505, -0.5794260043229126, 0.6305287521733175, 0.29932445515377715]
multi_threading:
  num_cost_threads: 4
  num_cnt_threads: 4


    
    
